javascript/javascript1/week4/getReply.js

```diff
@@ -0,0 +1,103 @@
+let userName = " ";
+let todo = [];
+function getReply(command) {
+  //'ask for name.'
+  const array = command.split(" ");
+  let response = " ";
+  if (command.includes("hello my name is ")) {
+    userName = array[array.length - 1];
+    return "Nice to meet you " + userName;
+  } else if (command.includes("what is my name")) {
+    return "your name is " + userName;
+  } else if (command.includes() {
+    return "your name is not find";
+  }
+  //'adding to todos.'
+  else if (command.includes("add ") && command.includes(" to my todos")) {
+    todo.push("fishing");
+    todo = command(command.length - 1);
+    return "fishing is added to your todos";
+  } else if (command.includes("add signging")) {
```

Comment:
You had a spelling error here "signging" instead of "singing", and again there is an issue with logging out with uppercase "add".
If you want a message to appear as well, as you commented after your console log, then don't forget to add a return statement with the message.